Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.8.7"
 [deps]
 Crayons = "a8cc5b0e-0ffa-5ad4-8c14-923d3ee1735f"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+SnoopPrecompile = "66db9d55-30c0-4569-8b51-7e840670fc0c"
```

Comment:
Will need a compat bound. We won't be integrating into the stdlib in the immediate future, that's still the goal, but more of a long-term one so an extra dep is fine.

Reply:
There's two registered versions, 1.0.0 and 1.0.1 ([src])
I can't find those two commit hashes in the repo, so no idea what's changed. I'll put "1" ig.

[src]: https://github.com/JuliaRegistries/General/blob/master/S/SnoopPrecompile/Versions.toml

Reply:
Yeah, just a `1` is fine.